projects/widgets/docs/inleiding.md

```diff
@@ -4,13 +4,9 @@
 
 <br>
```

Comment:
Btw this `<br>` should not be necessary (it adds some extra space between the logo and the header underneath but it also looks fine without it)